src/components/component.ts

```diff
@@ -401,30 +393,25 @@ module Plottable {
     }
 
     /**
-     * Checks if the Component has a fixed width or false if it grows to fill available space.
+     * Checks if the Component has a fixed width or if it grows to fill available space.
```

Comment:
Maybe change to
"Checks if the Component has a fixed width.
A component has a fixed width if it does not grow to fill the available width." or similar